tests/test_interconnect/test_interconnect_cgra.py

```diff
@@ -437,3 +438,143 @@ def test_interconnect_sram(cw_files, add_pd, io_sides):
                                magma_output="coreir-verilog",
                                directory=tempdir,
                                flags=["-Wno-fatal"])
+
```

Comment:
Can you remove the `add-pd` and make it always True?

Reply:
done.